### PR TITLE
Update maintainers list

### PR DESCRIPTION
**Pull request by JelleZijlstra**
I removed a few people who have not been active in the typing area for a long time, and added @rchen152 who is active on the repo and has commit rights. There are a few other people who have access to the repo but are not active. Listing them here does not seem useful.

### CODE REVIEW

**Comment by srittau** [comment]
Maybe we should add an "alumni" section at the bottom?

**Comment by JelleZijlstra** [comment]
Done. I omitted the other people who have access in case they had some reason to not want to be listed here.

**Review by srittau — APPROVED** [review]
LGTM, although I'd prefer to wait for @rchen152's approval.

**Review by rchen152 — APPROVED** [review]
Wow, didn't realize Matthias was still listed as a maintainer! (He was the original pytype TL.)

**Pull request merged**
